Assets/Scripts/GpsSensor.cs

```diff
@@ -51,21 +56,35 @@ private IEnumerator StreamPosition()
 
     private void ReportPosition()
     {
-        // Use double precision since geographic degrees are very large.
-        double latitude = CartesianToGeographic(transform.position.z + _noise * Utilities.GaussianRandom());
-        double longitude = CartesianToGeographic(transform.position.x + _noise * Utilities.GaussianRandom());
+        // Assuming rover initializes facing north, i.e. z+ (forward) is north & x+ (right) is east
+        double[] GPS = metersToGPS(new double[] {
+            transform.position.z + _noise * Utilities.GaussianRandom(),
+            transform.position.x + _noise * Utilities.GaussianRandom()});
```

Comment:
How does this assume the initial orientation of the rover? Seems like it's just using information about the unity coordinate system, and we've defined north to be +z. So I'd say just leave a comment saying that +z is north and +x is east.